pdb/src/main/java/org/cbioportal/pdb_annotation/scripts/PdbAnnotationScriptsPipelineStarter.java

```diff
@@ -0,0 +1,577 @@
+		}
+
+
+		public Integer getLength() {
+			return length;
+		}
+
+
+		public void setLength(Integer length) {
+			this.length = length;
+		}
+
+
+		public Integer getMismatch() {
+			return mismatch;
+		}
+
+
+		public void setMismatch(Integer mismatch) {
+			this.mismatch = mismatch;
+		}
+
+
+		public Integer getGapopen() {
+			return gapopen;
+		}
+
+
+		public void setGapopen(Integer gapopen) {
+			this.gapopen = gapopen;
+		}
+
+
+		public Integer getqStart() {
+			return qStart;
+		}
+
+
+		public void setqStart(Integer qStart) {
+			this.qStart = qStart;
+		}
+
+
+		public Integer getqEnd() {
+			return qEnd;
+		}
+
+
+		public void setqEnd(Integer qEnd) {
+			this.qEnd = qEnd;
+		}
+
+
+		public Integer getsStart() {
+			return sStart;
+		}
+
+
+		public void setsStart(Integer sStart) {
+			this.sStart = sStart;
+		}
+
+
+		public Integer getsEnd() {
+			return sEnd;
+		}
+
+
+		public void setsEnd(Integer sEnd) {
+			this.sEnd = sEnd;
+		}
+
+
+		public Double getEvalue() {
+			return evalue;
+		}
+
+
+		public void setEvalue(Double evalue) {
+			this.evalue = evalue;
+		}
+
+
+		public Double getBitscore() {
+			return bitscore;
+		}
+
+
+		public void setBitscore(Double bitscore) {
+			this.bitscore = bitscore;
+		}
+
+
+		@Override
+		public String toString(){
+			return oligoID.toString()+"\t"+qseqid+"\t"+sseqid+"\t"+pident+"\t"+length+"\t"+mismatch+"\t"+gapopen+"\t"+qStart.toString()+"\t"+
+					qEnd.toString()+"\t"+sStart.toString()+"\t"+
+					sEnd.toString()+"\t"+evalue.toString()+"\t"+
+					bitscore.toString();
+		}
+		
+	}
+	
+	
+	void test(){
+		try {
+
+            Structure s = StructureIO.getStructure("4cof");
+            
+            for(DBRef db : s.getDBRefs()){
+            	System.out.println("**");
+            	System.out.println(db.getChainId());
+            	System.out.println(db.getDatabase());
+            	System.out.println(db.getDbAccession());
+            	System.out.println(db.getDbIdCode());
+            	System.out.println(db.getInsertBegin());
+            	System.out.println(db.getInsertEnd());
+            	System.out.println(db.getDbSeqBegin());
+            	System.out.println(db.getDbSeqEnd());
+            	System.out.println(db.getIdbnsBegin());
+            	System.out.println(db.getIdbnsEnd());
+            	System.out.println(db.getIdCode());
+            	System.out.println(db.getId());
+            }
+
+            for ( Chain c : s.getChains()) {
+            	
+            	System.out.println(c.getChainID()+"\t"+c.getAtomLength()+"\t"+c.getAtomSequence().length()+"\t"+c.getSeqResLength());
+
+                // only the observed residues
+                System.out.println(c.getAtomSequence());
+                              
+
+                // print biological sequence
+                System.out.println(c.getSeqResSequence());
+            }
+
+        } catch (Exception e) {
+
+            e.printStackTrace();
+        } 
+	}
+	
+	
+    public static void main( String[] args )
```

Comment:
I have some problems in "mvn package" of the original project. It should get a war file and should be deployed under tomcat ? I put main() here for running the codes, even it has problems in integrating all the project together.